server/apps/spring-app/src/main/resources/mailetcontainer.xml

```diff
@@ -354,6 +354,11 @@ Regards, Postmaster XXX.YYY
           <gateway>otherserver.mydomain.com</gateway>
           <gatewayPort>25</gatewayPort>
             -->
+          <!-- in case of multiple gateway round-robin load balancing can bi activated -->
+          <!-- with the loadBalancing flag, default is false -->
```

Comment:
```suggestion
          <!-- with the loadBalancing flag, default is false -->
```

I think default can be true: this is definitly a desirable behaviour.

Reply:
alright its true now